### DISCUSSION TITLE
Not installed

### DISCUSSION THREAD

**Discussion by Kathxy** [thatskymod/Sky-CotL-Scripts#96] [Q&A]
Hi, i just downloaded Canva and sky and added the mods. If i play on play onto the app canva it displays that i havent installed sky the app, What am i supposed to do after adding the mods? If i also just go into sky normaly, no mods will show up there.

**Comment by thatskymod** [comment]
In Canvas, selecting the correct game version is important. Canvas supports three versions at the moment:

- **Live:** The global live version from Google Play, corresponding to the package `com.tgc.sky.android`.
- **Beta:** The global beta version from Google Play, also referred to as `com.tgc.sky.android.test.gold`.
- **Huawei:** The Huawei AppGallery version of Sky, identified as `com.tgc.sky.android.huawei`.

Other versions are not currently supported.

Have you confirmed that the version you selected aligns with your preferences and the correct package is installed?

Otherwise, this issue might stem from a software conflict when querying the package list, potentially related to the device or its firmware.

**Comment by thatskymod** [comment]
Explanation provided in https://github.com/thatskymod/Sky-CotL-Scripts/discussions/96#discussioncomment-9884544

To offer further help, we would need more details about your device and environment, which were not provided.

Closing this due to the lack of inactivity.
